pkg/controller/ibmblockcsi/syncer/csi_controller.go

```diff
@@ -134,10 +135,14 @@ func (s *csiControllerSyncer) ensureContainersSpec() []corev1.Container {
 	})
 
 	// csi provisioner sidecar
+	provisionerArgs := []string{"--csi-address=$(ADDRESS)", "--v=5", "--timeout=30s", "--default-fstype=ext4"}
+	if TopologyEnabled {
+		provisionerArgs = append(provisionerArgs, "--feature-gates=Topology=true")
+	}
 	provisioner := s.ensureContainer(provisionerContainerName,
 		s.getCSIProvisionerImage(),
 		// TODO: make timeout configurable
```

Comment:
Do we need this comment here?

Reply:
I dont know, I was not the one that put it

Reply:
You moved the reference. I think we should remove it or move it to the relevant line

Reply:
I would move it to the relevant line for now, because it exists in several places, which are not relevant for this ticket

Reply:
done